auction9-frontend/src/main.ts

```diff
@@ -20,6 +18,9 @@ Amplify.configure({
     // REQUIRED - Amazon Cognito Region
     region: 'eu-west-1',
 
+    //REQUIRED - Amazon Cognito Identity Pool ID
+    identityPoolId: 'eu-west-1:708b1637-6afc-4698-b5a0-6c8f2cd3b52e',
```

Comment:
Should be part of configuration. Along with other configuration below.

Reply:
Fixed